betelgeuse.py

```diff
@@ -814,3 +814,134 @@ def test_run(
     pool.close()
     pool.join()
     TestRun.session.tx_commit()
+
+
+def create_xml_property(name, value):
+    """Create an XML property element and set its name and value attributes."""
+    element = ElementTree.Element('property')
+    element.set('name', name)
+    element.set('value', value)
+    return element
+
+
+@cli.command('xml-test-run')
+@click.option(
+    '--custom-fields',
+    help='Indicates to the importer which custom fields should be set. '
+    'Expected format is either id=value or JSON format {"id":"value"}. This '
+    'option can be specified multiple times.',
+    multiple=True,
+)
+@click.option(
+    '--dry-run',
+    help='Indicate to the importer to not make any change.',
+    is_flag=True,
+)
+@click.option(
+    '--lookup-method',
+    default='custom',
+    help='Indicates to the importer which lookup method to use. "id" for work '
+    'item id or "custom" for custom id (default).',
+    type=click.Choice([
+        'id',
+        'custom',
+    ])
+)
+@click.option(
+    '--not-finished',
```

Comment:
The reason why `--not-finished` was chosen is because the default importer is to set to finished so makes more sense to reproduce that here since it would be the default choice. Most of the time we are importing the test-run result that are already completed and makes sense to set to finished by default.

Using `--finished` instead will require the users to set this option most of the time. To me the best usability is to have a negative flag that will disable the default behavior.